loader/src/injector/hook.cpp

```diff
@@ -617,17 +637,20 @@ void ZygiskContext::fork_pre() {
 }
 
 void ZygiskContext::sanitize_fds() {
-    if (flags[SKIP_FD_SANITIZATION])
+    if (flags[SKIP_FD_SANITIZATION]) {
         return;
+    }
 
     if (flags[APP_FORK_AND_SPECIALIZE]) {
         auto update_fd_array = [&](int off) -> jintArray {
-            if (exempted_fds.empty())
-                return nullptr;
+            if (exempted_fds.empty()) {
+                return NULL;
+            }
 
             jintArray array = env->NewIntArray(static_cast<int>(off + exempted_fds.size()));
-            if (array == nullptr)
-                return nullptr;
+            if (array == NULL) {
```

Comment:
Ditto (Okay, I'm tired of listing them all, if anything I can mark the rest later on)